Dockerfile

```diff
@@ -2,10 +2,11 @@ FROM debian:stretch
 
 # Install additional Debian packages
 RUN apt-get -y update && apt-get upgrade -y && apt-get install -y \
-        vim python python-pip \
+        vim python python-pip python-dev\
         openssl userv openssh-client \
         apache2 apache2-utils libapache2-mod-wsgi \
-        libssl-dev libjpeg-dev netcat zlib1g-dev
+        libssl-dev libjpeg-dev netcat zlib1g-dev \
+        libpq-dev build-essential
 
```

Comment:
we should remove all the dev packages after we have executed pip install, that will slim the image and will contain less unnecessary things.